example-form/pages/app/businesses/index.tsx

```diff
@@ -1,11 +1,16 @@
-import { Fragment, ReactElement } from 'react';
+import { Fragment, ReactElement, useEffect } from 'react';
 import { PageContent } from '@ag.ds-next/react/content';
 import { H1 } from '@ag.ds-next/react/heading';
 import { DocumentTitle } from '../../../components/DocumentTitle';
 import { AppLayout } from '../../../components/Layout/AppLayout';
 import type { NextPageWithLayout } from '../../_app';
 
 const Page: NextPageWithLayout = () => {
+	useEffect(() => {
```

Comment:
If you go to the businesses page in the PR preview, you can this error boundary in action.